<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>338</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>338</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textArea">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 italic 14pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="styleSheet">
        <string notr="true">font: 75 italic 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="styleSheet">
        <string notr="true">font: 75 italic 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteButton">
       <property name="styleSheet">
        <string notr="true">font: 75 italic 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Paste</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cutButton">
       <property name="styleSheet">
        <string notr="true">font: 75 italic 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Cut</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="styleSheet">
        <string notr="true">font: 75 italic 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="openPath">
       <property name="styleSheet">
        <string notr="true">font: 75 italic 12pt &quot;微软雅黑&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="styleSheet">
        <string notr="true">font: 75 italic 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="savePath">
       <property name="styleSheet">
        <string notr="true">font: 75 italic 12pt &quot;微软雅黑&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
